core/src/epicli/cli/helpers/objdict_helpers.py

```diff
@@ -27,14 +28,53 @@ def objdict_to_dict(something):
     return _nested_dict_to_dict(something, src_class=dict, dst_class=dict)  # ObjDict is a subclass of dict
 
 
+def is_named_list(l):
+    count = select_all(l, lambda x: hasattr(x, 'name'))
+    return len(count) == len(l)
```

Comment:
One-liner alternative:
```suggestion
    return all(hasattr(x, 'name') for x in l)
```

Reply:
Nice! will test and apply!